gkeepapi/__init__.py

```diff
@@ -418,11 +444,20 @@ def load(self, auth, state=None, sync=True):
         """
         self._keep_api.setAuth(auth)
         self._reminders_api.setAuth(auth)
+        self._auth = auth
         if state is not None:
             self.restore(state)
         if sync:
             self.sync(True)
 
+    def getAuth(self):
```

Comment:
Not sure if we need to expose this n store it locally, since we'll never have to write to it outside initial bootstrap.